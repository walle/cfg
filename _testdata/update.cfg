# Test data for verifying that updating a config works

# string value (string)
string = bar

# integer value (int)
integer = 42

# float value (float)
float = 4.2

# boolean value (bool)
boolean = false
