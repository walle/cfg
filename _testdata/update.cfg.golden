# Test data for verifying that updating a config works

# string value (string)
string = foobar

# integer value (int)
integer = 404

# float value (float)
float = 3.14

# boolean value (bool)
boolean = true
